Framework/Geometry/src/Rendering/GeometryHandler.cpp

```diff
@@ -75,17 +75,55 @@ size_t GeometryHandler::numberOfPoints() const {
   return 0;
 }
 
-const std::vector<double> &GeometryHandler::getTriangleVertices() const {
+const std::vector<double> GeometryHandler::renderCuboid(const detail::ShapeInfo &shapeInfo) {
+  const auto &points = shapeInfo.points();
```

Comment:
I wonder if we should implement renderCuboid and all store m_MeshPoint and m_meshVertices in the handler class. It looks like the handler is providing an interface for the rendering, rather than implementing it. Maybe this should be separated out into another class

Reply:
This could be deleted as this rendering can already be handled by OpenCascade in `RenderingHelpersOpenGL.cpp`.